updatecli/updatecli.d/charts/jenkins.yaml

```diff
@@ -13,12 +27,17 @@ targets:
       file: clusters/prodpublick8s.yaml
       matchPattern: 'chart: jenkins\/jenkins((\r\n|\r|\n)(\s+))version: .*'
       replacePattern: 'chart: jenkins/jenkins${1}version: {{ source `lastChartVersion` }}'
-    scm:
-      github:
-        user: "{{ .github.user }}"
-        email: "{{ .github.email }}"
-        owner: "{{ .github.owner }}"
-        repository: "{{ .github.repository }}"
-        token: "{{ requiredEnv .github.token }}"
-        username: "{{ .github.username }}"
-        branch: "{{ .github.branch }}"
+    scmID: default
+
+pullrequests:
+  default:
+    kind: github
+    scmID: default
+    targets:
+      - updateChartVersion
+    spec:
+      labels:
+        - dependencies
+        - infra.ci.jenkins.io
+        - release.ci.jenkins.io
```

Comment:
The idea was to have labels with the domain name (since this charts is used for different apps). 

As we discussed together, let's define labels as it:
- `chart:<chart short name>`
- `site:<DNS>`